hydra-node/test/Hydra/NodeSpec.hs

```diff
@@ -105,9 +107,9 @@ spec = parallel $ do
       let events =
             [ NetworkEvent{ttl = defaultTTL, message = Connected{nodeId = NodeId "NodeId1"}}
             , NetworkEvent{ttl = defaultTTL, message = Connected{nodeId = NodeId "NodeId2"}}
-            , ClientEvent $ Init cperiod
```

Comment:
Is `cperiod` still used?

Reply:
This must be an old diff since I removed `cperiod` from `Init` already.